Automattic/themes
# Fix course theme comment styles

#### Changes proposed in this Pull Request:

Fixes discrepancies in the comment thread and reply form styles among different templates, like posts, page, lesson, and optional templates.

### Testing instructions

1. Enable the Course theme
2. Enable Sensei
3. Create a page and a post, make sure to enable comments
4. Create a Course with a lesson in sensei, and enable comments for it too
5. Now visit the page and post from the frontend as a logged out user
6. Make a few comments
7. Make sure the thread and the reply forms look close to the design
8. Now open the post in the editor
9. Select each custom template from the right dropdown, like Single Column - Featured Image Only or Single Column - Title only, etc.
10. After selecting each of them, click on update and check from the frontend. Make sure the comments are not broken in any of them.
11. Now check them again as logged in user.
12. You won't see all the fields now, but make sure it doesn't appear broken.
13. Make sure to check for the lesson now too. Add a couple of comments, make sure the thread looks like the other ones from page and post type.

#### For now, disregard if you see the comment portion getting duplicated in lessons. It'll be fixed in a separate PR under a different issue. Only check for the comment portion at the bottom, not the top one.

We tried reusing the existing spacing and size presets to not introduce new ones. So in some places, the spacings are few pixels different, but they don't fall far from the design. Also, the design has some spacings that are not uniform, like the spacing between all the input fields or between the input fields and their labels, like the spacings between the email label and its input field and the comment label and its input field. We used a uniform spacing in these places.

![image](https://github.com/Automattic/themes/assets/6820724/d26c0d1e-249f-4eee-97cd-3efd3ec0d276)


### Screenshots

Desktop:
![image](https://github.com/Automattic/themes/assets/6820724/5a911f98-5641-4e98-9365-c33a8fd080de)

Mobile:
![image](https://github.com/Automattic/themes/assets/6820724/6192dd6c-1da5-4ac8-a1d8-351088c9077d)

Design:
![image](https://github.com/Automattic/themes/assets/6820724/970178be-3909-41d9-81f7-8625fdc09fc5)

#### Related issue(s):

https://github.com/Automattic/themes/issues/6977
